lib/src/v3/connection.dart

```diff
@@ -603,6 +603,15 @@ class _PgResultStreamSubscription
         _affectedRows.complete(message.rowsAffected);
       case ReadyForQueryMessage():
         await _completeQuery();
+      case CopyBothResponseMessage():
+        // This message indicates a successful start for Streaming Replication.
+        // Hence, in this context, the query is complete. And from here on,
+        // the server will be streaming replication messages.
+        // But if the connection was used after this point to execute further
+        // queries, the server messages will be blocked.
+        // TODO(osaxma): Prevent executing queries when Streaming Replication
+        //               is ongoing
+        await _completeQuery();
```

Comment:
This is a temporary special case for Streaming Replication since `CopyBothResponseMessage` is only used for that. Ideally, we should stop any queries from being executed after this point to prevent blocking the server messages. 

I will try updating the example repository for Steaming Replication to investigate what could go wrong and how to better handle this.